### ISSUE TITLE
Unexpected exception raised in websocket

### ISSUE THREAD

**Issue by haohedev** [hasura/graphql-engine#4344]
![image](https://user-images.githubusercontent.com/5556270/78859559-40287b80-7a62-11ea-8f16-3d5f5f16db67.png)

**Comment by haohedev** [comment]
The Hasura version is v1.1.1

**Comment by tirumaraiselvan** [comment]
@haohedev Thanks for the report.

Is there any repro you can provide?

**Comment by haohedev** [comment]
![image](https://user-images.githubusercontent.com/5556270/78897134-4c352d00-7aa4-11ea-84c3-47c70edd5f5e.png)

Before April 2nd, the version I was using was 1.1.0. It can be seen from the chart that there will be a memory leak, but the container will automatically restart. This is similar to the #4077 problem. After I used version 1.1.1, it seems that the problem is solved. But today a process of containers turning into zombies emerged. Now I added a liveness probe, I hope it works.
The scenario used by Hasura is the subscription data change, which is displayed on the front end. There will be programs that write data directly to the database.
Sorry for my english, i use google translate
